MinecraftClient/ChatBot.cs

```diff
@@ -328,9 +333,16 @@ internal void ProcessQueuedText()
         /// <summary>
         /// Called when the health of an entity changed
         /// </summary>
-        /// <param name="entityID">Entity ID</param>
+        /// <param name="entity">Entity</param>
         /// <param name="health">The health of the entity</param>
-        public virtual void OnEntityHealth(int entityID, float health) { }
+        public virtual void OnEntityHealth(Entity entity, float health) { }
+
+        /// <summary>
+        /// Called when the metadata of an entity changed
+        /// </summary>
+        /// <param name="entity">Entity</param>
+        /// <param name="metadata">The metadata of the entity</param>
+        public virtual void OnEntityMetadata(int entityID, Dictionary<int, object> metadata) { }
```

Comment:
Should be `OnEntityMetadata(Entity entity [...]` for consistency with other APIs